AmazonChimeSDK/AmazonChimeSDK.xcodeproj/project.pbxproj

```diff
@@ -734,6 +725,14 @@
 			path = utils;
 			sourceTree = "<group>";
 		};
+		C6C2F104250BD96F001678B8 /* Recovered References */ = {
+			isa = PBXGroup;
+			children = (
+				1C38DDD124EC0DBA005524AB /* MockVideoClientController.swift */,
+			);
+			name = "Recovered References";
+			sourceTree = "<group>";
+		};
```

Comment:
Delete these?

Reply:
Removed. See the other comment for more detail.